src/components/databaseUi/CounterStoreUI.tsx

```diff
@@ -0,0 +1,77 @@
+import DatabaseUIProvider from "../../providers/DatabaseUIProvider";
+import React, { useState } from 'react';
+import ToolbarStyle from '../viewDatabase/Sidebar.module.css';
+import { Store } from "orbit-db-store";
+import CounterStoreStyles from './CounterStoreUI.module.css';
+
+export default class CounterStoreUI implements DatabaseUIProvider {
+
+  getSidebar: React.FC<Store> = ({ store }) => {
+    const [input, setInput] = useState("");
+    const [activeField, setActiveField] = useState("");
+    
+    function handleFormSubmit(e: React.FormEvent<HTMLFormElement>) {
+      e.preventDefault();
+      let amount = Number(input);
+      store.inc(amount);
+    }
+
+    const incrementField = (opName, op) => {
+      return () => (
+        <div className={ToolbarStyle.inputFieldContainer}>
+          <form onSubmit={(e) => {e.preventDefault(); op(e)}}>
```

Comment:
You do not need the e parameter as you only use it to prevent default which is done in the onsubmit already